docs/source/orderer/ordering_service.md

```diff
@@ -178,10 +185,10 @@ ledger L1 on P2. Once this process is complete, the ledger L1 has been
 consistently updated on peers P1 and P2, and each may inform connected
 applications that the transaction has been processed.*
 
-In summary, phase three sees the blocks generated by the ordering service applied
-consistently to the ledger. The strict ordering of transactions into blocks
-allows each peer to validate that transaction updates are consistently applied
-across the blockchain network.
+In summary, phase three sees the blocks of transactions created by the ordering
+service applied consistently to the ledger by the gateway. The strict
```

Comment:
The ordered blocks are not returned to gateway.
The ordered blocks are distributed to all peers.
The gateway listens for peer transaction commit event before responding back to client.

Reply:
ok.